cdc/changefeed.go

```diff
@@ -521,15 +533,16 @@ func (c *changeFeed) calcResolvedTs(ctx context.Context) error {
 		minCheckpointTs = minResolvedTs
 	}
 
+	c.status.ResolvedTs = minResolvedTs
+	c.status.CheckpointTs = minCheckpointTs
```

Comment:
these two lines have conflict with L541 and L545. `c.sink.EmitCheckpointEvent` will be never called?